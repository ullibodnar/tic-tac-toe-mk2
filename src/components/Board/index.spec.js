import React from 'react'
import { shallow } from 'enzyme'

import Board from '.'

describe('components:Board', () => {
  it('renders the Board with the proper stylesm', () => {
    expect(toJson(shallow(<Board />))).toMatchSnapshot()
  })
})
