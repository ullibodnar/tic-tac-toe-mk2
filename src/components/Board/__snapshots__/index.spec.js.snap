// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:Board renders the Board with the proper stylesm 1`] = `
.c0 {
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  display: grid;
  grid-area: board;
  grid-gap: 0;
  grid-template-areas: 'zero one two' 'three four five' 'six seven eight';
  grid-template-columns: 20vh 20vh 20vh;
  grid-template-rows: 20vh 20vh 20vh;
  height: 60vh;
  justify-self: center;
  margin: auto;
  width: 60vh;
}

<div
  className="c0"
/>
`;
